types/chrome/index.d.ts

```diff
@@ -167,7 +167,7 @@ declare namespace chrome.action {
      * @param tabId The id of the tab for which you want to modify the action.
      * @param callback
      */
-    export function disable(tabId?: number, callback?: () => void): void;
+    export function disable(tabId?: number, callback: () => void): void;
```

Comment:
Reviewer note: These changes were made only on methods that already had a Promise-returning definition.